projects/lights_control/inc/lights_can.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "can.h"
+#include "lights_events.h"
+#include "lights_gpio.h"
+
+#define CAN_NUM_RX_HANDLERS 1
+
+// initializes the lights_can module.
+// needs to know whether it's the front light board or
+// the rear light board.
+void lights_can_init(BoardType type, bool loopback);
+
+// This will get used by the sync module
+// Rear board is the board that will be sending the
+// sync message to the front board. It raises a sync
+// event at the current board, and it transmits a CAN
+// sync message.
+
+StatusCode send_sync(void);
```

Comment:
Move this to like `lights_sync.h` or something? It's okay if it doesn't actually implement the sync logic yet, but that separation would be nice, especially if you're writing a new module for it.

Reply:
Oh true, my original philosophy was to keep this module as the only module that raises events and deals with CAN and that all other modules use this module when they need to deal with something CAN related. But I feel like the separation here would make sense. There's only one concern that `lights_sync` would be dependent on this module being initialized first, otherwise `can_transmit()` won't work there. But I guess that's fine 👍

Reply:
In general we prefer to defer module initialization to the application layer if a module is used in more than one module. Then you won't have an ordering problem.